generation/2d_ddpm/2d_ddpm_inpainting.ipynb

```diff
@@ -0,0 +1,757 @@
+{
```

Comment:
<p>I guess here you mean from the generative repo? May need mention it here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1814/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>This is something that should be changed to reference the distributed training tutorial here in the mail repo once this is ported from GenerativeModels</p><p> </p>